internal/provider/waypoint/data_source_waypoint_application_template.go

```diff
@@ -123,6 +124,27 @@ func (d *DataSourceApplicationTemplate) Schema(ctx context.Context, req datasour
 					},
 				},
 			},
+			"variable_options": schema.ListNestedAttribute{
+				Optional:    true,
```

Comment:
All of this should be `computed` for the data source unless it is identifying information used to retrieve the data.